server/libs/job/src/constants/queue-name.constant.ts

```diff
@@ -3,3 +3,10 @@ export const assetUploadedQueueName = 'asset-uploaded-queue';
 export const metadataExtractionQueueName = 'metadata-extraction-queue';
 export const videoConversionQueueName = 'video-conversion-queue';
 export const generateChecksumQueueName = 'generate-checksum-queue';
+
+export enum QueueNameEnum {
```

Comment:
Can't we refactor the existing consts into an enum and use those here? This feels like a bunch of duplication atm.

Reply:
I did try that but Typescript doesn't allow you to use computed values
<img width="779" alt="Screen Shot 2022-10-02 at 09 53 58" src="https://user-images.githubusercontent.com/27055614/193460582-38c15651-8a01-4e40-965e-486fe662cea6.png">

Reply:
What about replacing the `videoConversionQueueName` etc with an enum and using that value in both places?

Reply:
Make sense, I will replace this used values around the code based as well.